src/main/java/mat/model/clause/MeasureShareDTO.java

```diff
@@ -418,6 +420,25 @@ public String getReleaseVersion() {
 	public void setReleaseVersion(String releaseVersion) {
 		this.releaseVersion = releaseVersion;
 	}
+
+	/**
+	 * Gets the qdmVersion.
+	 *
+	 * @return the qdmVersion
+	 */
+	public String getQdmVersion() {
+		return qdmVersion;
+	}
+
+	/**
+	 * Sets the qdmVersion.
+	 *
```

Comment:
I believe we don't need auto-generated comments. It just clutters the code.

Reply:
Do I have to remove comments for entire MeasureShareDTO class ?

Reply:
I believe so. Why would you need any pointless comments.. 

https://www.todaysoftmag.com/article/1120/clean-code-comments-and-formatting

Reply:
Done